AEXS/aex-4.md

```diff
@@ -48,16 +48,29 @@ This is especially beneficial for mobile wallet users as they can simply click o
 
 2. Payment
 
-    URI: `aeternity:<receiver_address>/<amount>/<vendor_identifier>?callback=<url>&txId=<transaction_id>`
+    URI: `aeternity:<receiver_address>/<amount>/<vendor_identifier>?callback=<url>`
 
     This URI scheme enables vendors to generate direct payment request URIs to user wallet.
 
     1. Vendor generates a address for receiving payment.
-    2. Creates the URI substituting `receiver_address` with newly created address, `amount` with amount requested, `vendor_identifier` with a human readable identifier that can be used to correctly identifiy vendor and requested payment, and `url` with a callback URL that accepts a transaction id under `txId` query param.
+    2. Creates the URI substituting `receiver_address` with newly created address, `amount` with amount requested, `vendor_identifier` with a human readable identifier that can be used to correctly identify vendor and requested payment, and `url` with a callback URL that accepts a transaction id under `txId` query param.
 
     #### Example
 
-    `aeternity:ak_2gUJrd11cy65yqZ7mg1ULUG4kZ5r6v6vNqVtmA8HqUGKCf6kNf/125.12/myaeshop-ref9834?callback=https://myaeshop.com/verify?txId=th_rox6XyzqDBTopoTYQNJux49jnLVd39QhxDSvYoQ5he9SAGupz`
+    `aeternity:ak_2gUJrd11cy65yqZ7mg1ULUG4kZ5r6v6vNqVtmA8HqUGKCf6kNf/125.12/myaeshop-ref9834?callback=https://myaeshop.com/verify`
+
+3. Sign and Broadcast
+
+    URI: `aeternity:<transaction>/<network_id>/<label>?return=<txId/tx>&callback=<url>`
+
+    This URI scheme enables users to sign(and broadcast) the transaction using deep lining enabled user wallet.
+
+    1. Vendor generates a address for receiving payment.
+    2. Creates the URI substituting `transaction` and `network_id` with their respective value, `label` a human readable text regarding the transaction, `return` which indicates the return value in the callback URL where `txId` means the wallet needs to broadcast the transaction and return the transaction id whereas `tx` means that the callback url expects a signed transaction instead, and at last a `url` with a callback URL that accepts a transaction id under `txId` query param and signed transaction output under `tx` query param.
```

Comment:
Maybe you can make it clearer that you mean the literal strings `"txId"` and `"tx"`, because you basically use the same notation for placeholders. There's an example but just to be sure :)

Reply:
ok. makes sense. I'll add another section describing all the parameters with the datatype.